src/components/sponsors/sponsor-group.js

```diff
@@ -22,7 +22,7 @@ function SponsorGroup({sponsors, title}) {
       {
         chunkedSponsors.map((rowSponsors, index) => {
           return (
-            <div className={`${css(styles.group)} +space-children`} key={index}>
+            <div className={css(styles.group)} key={index}>
```

Comment:
are you sure nothing changes by removing this class? I would expect the sponsors get closer together. Could you please provide a screenshot? Thanks!